lifecycle_policy.md

```diff
@@ -82,7 +82,7 @@ Once a project has submitted a proposal as outlined above, the process for accep
 
    These rules apply for all criteria on the ballot:
 
-   * If the number of votes cast (including explicit "Abstain" votes) is less than half of the number of voting TAC members, then the result of the poll is "Against".
+   * If the number of votes cast (including explicit "Abstain" votes) is not greater than half of the number of voting TAC members, then the result of the poll is "Against".
    * Otherwise, if the number of "Approve" votes is at least two thirds of the votes cast (not including explicit "Abstain" votes), then the result of the poll is "Approve".
    * Otherwise, the result of the poll is "Against".
```

Comment:
This looks like what we discussed in our meeting today. However, now that I read this, what is the function of the first condition? It looks like you could just remove it and get the same function. That is how is the following not different?

```suggestion
   * If the number of "Approve" votes is at least two thirds of the votes cast (not including explicit "Abstain" votes), then the result of the poll is "Approve".
   * Otherwise, the result of the poll is "Against".
```

Reply:
The issue is of quorum -- if only 3 people vote, it doesn't matter that they're unanimously in favor.